### DISCUSSION TITLE
feature request: import text files from disk

### DISCUSSION THREAD

**Discussion by psivesely** [standardnotes/forum#2499] [Suggestions] [answered]
Instead of having to copy and paste content into standard notes, it would be nice to be able to import text files directly.

**Comment by atmoio** [comment]
You can do this here: https://dashboard.standardnotes.org/tools

**Comment by psivesely** [comment]
feature request: build this functionality directly into the client.

**Comment by myreli** [comment, marked as answer]
Since `v3.131.0`, importing notes from multiple sources is now built-in in the client: 

<img width="1034" alt="image" src="https://user-images.githubusercontent.com/17554234/218580555-af77ff99-43bd-4148-a716-4f81fef6db24.png">
